src/plugins/intel_cpu/src/infer_request.cpp

```diff
@@ -343,12 +332,12 @@ std::vector<ov::SoPtr<ov::ITensor>> SyncInferRequest::get_tensors(const ov::Outp
 }
 
 const ov::Output<const ov::Node>& SyncInferRequest::get_internal_port(const ov::Output<const ov::Node>& port) const {
-    auto name = get_port_name(port);
```

Comment:
@riverlijunjie what is the different between internal port and port? I think they are the same

Reply:
After API 1.0 has been removed completely, it should be same. Please have some tests to confirm it.